frontend/src/exams/add-exam-form-components.js

```diff
@@ -182,8 +188,8 @@ export const ExamReceivedQuestion = Vue.component('exam-received-question', {
             <span v-if="error" style="color: red">{{ validationObj[q.key].message }}</span>
           </label><br>
           <b-form-radio-group v-model="showRadio"
-                              @input="preSetDate"
-                              :options="addExamModal.setup === 'other' ? otherOptions : options" />
+                              @input="preSetDate()"
+                              :options="modalSetup == 'other' || modalSetup == 'pesticide' ? otherOptions : options" />
```

Comment:
Should this be `===`?

Reply:
i think for awhile i had a more complex condition in that and it was 1 single character past the 120 character mark.  however, i made it simpler and so yes, i should have reverted to using the strict equality operator.  will fix.